pkg/client/dtclient/settings_client.go

```diff
@@ -707,6 +722,31 @@ func buildPostRequestPayload(ctx context.Context, remoteObjectId string, obj Set
 	return dest.Bytes(), nil
 }
 
+// insertAfterToPayloadValue converts the insertAfter value to the propler
+// value required in the payload.
+//
+// For POST (only method that we use), it must be as follows:
+//
+//   - insert to the front: `insertAfter` to ”
+//   - insert to the back: `insertAfter` to nil (default behavior of monaco)
+//   - insert after another item: `insertAfter` to the ID of the item
+func insertAfterToPayloadValue(insertAfter *string) *string {
+
+	if insertAfter == nil {
+		return nil
+	}
+
+	switch *insertAfter {
+	case InsertPositionBack:
+		return nil
+	case InsertPositionFront:
+		var empty = ""
+		return &empty
+	default:
+		return insertAfter
+	}
+}
+
 // parsePostResponse unmarshalls and parses the settings response for the post request
```

Comment:
While I see it, considering the spelling of the function:
```suggestion
// parsePostResponse unmarshals and parses the settings response for the post request
```